### ISSUE TITLE
getSources() not working in Chrome

### ISSUE THREAD

**Issue by marynotari** [processing/p5.js-sound#231]
Browser: Chrome Version 62.0.3202.94 (Official Build) (64-bit)

I'm currently working on a project that uses p5.AudioIn() to use live audio input for a sketch. I was going to experiment with two mic inputs and was trying to figure out how to connect them using `getSources()` and `setSource()`. In the p5 documentation it says that [getSources()](https://p5js.org/reference/#/p5.AudioIn/getSources) works only in Chrome but when I tried to [test it](https://alpha.editor.p5js.org/marynotari/sketches/rk6SbGzbG) using the web editor in Chrome, I get this error message: 

```
This browser does not support MediaStreamTrack.getSources(). 
```

```javascript
var audiograb;

function setup(){
  //new audioIn
  audioGrab = new p5.AudioIn();

  audioGrab.getSources(function(sourceList) {
    //print out the array of available sources
    console.log(sourceList);
    //set the source to the first item in the inputSources array
    audioGrab.setSource(0);
  });
}
```

With a little Googling I learned that Chrome had deprecated MediaStreamTrack.getSources(). My question: is there a work-around?

**Comment by therewasaguy** [comment]
looking into this, thanks for opening the issue @marynotari !

**Comment by therewasaguy** [comment]
should be fixed by #234
